en/lessons/advanced/nerves.md

```diff
@@ -0,0 +1,231 @@
+---
+version: 1.0.0
+title: Nerves
+---
+
+## Introduction and requirements
+
+We will be talking about Nerves in this lesson. Nerves project is a framework for using Elixir in embedded software development. As their site says, `Craft and deploy bulletproof embedded software in Elixir`. Due to the topic, this lesson will be a bit different from other Elixir School lessons. Nerves is an advanced topic and require some advanced system setup and additional hardware, as well as understanding what you are doing. You've been warned!
+
+Today we are going to be exploring this land of embedded development. So, if you have hardware from their list of [supported targets](https://hexdocs.pm/nerves/targets.html) , card reader with a memory card supported by the hardware of your choice + wired networking connection to access this device by the network, we are ready to start. However, we suggest you have a Raspberry PI, due to it having controllable LED onboard. Otherwise, you will need to build a scheme similar to [this one](https://github.com/fhunleth/elixir_ale#gpio) to make use of external LED. We also suggest you have a screen connected to your hardware of choice - this will simplify configuration a lot.
```

Comment:
PI -> pi.

It'd be kind of a pain, but for the sake of diffing some of us have been writing in a style of one-sentence-per-line for very friendly diffing as time goes on and edits come in. Might not be worth doing here, but figured I'd note.

Reply:
Thanks, fixed. 

Hm, I'll look into other lessons.

Reply:
I've checked, and more complex lessons like https://elixirschool.com/en/lessons/advanced/typespec/ and https://elixirschool.com/en/lessons/advanced/gen-stage/ use multiple sentences per paragraph due to information density. I guess this is fine here.